dotcom-rendering/src/web/components/App.tsx

```diff
@@ -320,96 +304,6 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 		}
 	}, [CAPI.shouldHideReaderRevenue]);
 
-	// kick off the CMP...
-	useOnce(() => {
-		if (!CAPI.config.switches.consentManagement) return; // CMP turned off!
```

Comment:
Do we replicate this functionality with the new implementation? (If needed???? Should we be able to turn CMP off????)

Reply:
If the switch exists, we should support it. Whether it should exist is another question.

@ashishpuliyel ?

Reply:
I confirm we should be able to turn the CMP off, for instance if there is an issue with Sourcepoint